DNN Platform/Library/Entities/Modules/Actions/ModuleGenericEventArgs.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Entities.Modules.Actions
+{
+    /// <summary>
+    /// Event arguments for a generic module event.
+    /// </summary>
+    public class ModuleGenericEventArgs : ModuleEventArgs
```

Comment:
Would we want to introduce a separate generic event for things that aren't related to a specific module?

Reply:
Probably. But it's good to have one for modules regardless of that effort, IMHO, given that we pass in the moduleinfo which is very rich in providing context.